### DISCUSSION TITLE
Pace Caret setting per mode

### DISCUSSION THREAD

**Discussion by mkozden** [monkeytypegame/monkeytype#968] [Ideas] [answered]
I think we should be able to change the pace caret setting individually for each mode. for example, when typing quotes, it's highly unlikely to get the same quote often, so you might want to use the personal best caret for that. But for other modes, using the average caret might be better.

**Comment by Miodec** [comment, marked as answer]
This is not fully relevant anymore because quote pb's have been removed.

**Reply by mkozden** [reply]
Aww, that sucks. Could you please tell me why you removed them?

**Reply by Miodec** [reply]
Quote pbs didnt make much sense because you dont type the same quote often (plus its going to get even more rare with people adding more quotes). Plus keeping track of a pb for every quote (about 6000 now) is a lot of data stored.

**Reply by mkozden** [reply]
Oh right, that makes sense. Thanks for answering.
